webapp/package.json

```diff
@@ -27,7 +27,7 @@
     "lodash": ">=4.17.11"
   },
   "dependencies": {
-    "mattermost-redux": "^5.9.0",
+    "mattermost-redux": "^5.10.0",
```

Comment:
Is it okay not to make this a fix version?

Reply:
It should be fine. This is actually the preferred way of using the package.json since the package-lock.json is supposed to handle the actual commit hash.